src/lib/components/error-pages/ErrorPageStyle.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { spacing, fontSize } from '../../style/theme';
+import { getThemePropSelector } from '../../utils';
+
+export const ErrorPageContainer = styled.div`
+  background-color: ${getThemePropSelector('backgroundLevel1')};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+`;
+
+export const Title = styled.h2`
+  color: ${getThemePropSelector('textPrimary')};
+  margin: 0;
```

Comment:
is the `margin: 0` useful here ?

Reply:
Yes :D 
I have a margin around the title because we styled `h2`. Let's keep the current style if it doesn't bother you too much?